create_monorepo.py

```
@@ -40,7 +41,7 @@ def lines(s):
 
 def git(args):
     print("+ git:", args)
-    return subprocess.check_output("git " + args, universal_newlines=True, shell=True)
+    return subprocess.check_output(["git"] + shlex.split(args), universal_newlines=True)
```

Ruff (0.15.20)

[error] `subprocess` call: check for execution of untrusted input

(S603)